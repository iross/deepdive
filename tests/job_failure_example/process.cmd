# MAKE SURE TO CHANGE THE FIRST SECTION BELOW FOR EACH NEW SUBMISSION!!!
#
# By default, your job will be submitted to the CHTC's HTCondor 
# Pool only, which is good for jobs that are each less than 24 hours.
#
# If your jobs are less than 4 hours long, "flock" them additionally to 
# other HTCondor pools on campus by uncommenting the below line:
#+WantFlocking = true
#
# If your jobs are less than ~2 hours long, "glide" them to the national
# Open Science Grid (OSG) for access to even more computers and the
# fastest overall throughput. Uncomment the below line:
#+WantGlidein = true
#
# Tell Condor how many CPUs (cores), how much memory (MB) and how much 
# disk space (KB) each job will need:
request_cpus = 1
request_memory = 1000
request_disk = 1000000



# SECOND SECTION: Unlikely that you'll need to change this each time
# The below lines indicate that all files necessary for each job
# need to be transfered to it. Only modify these if you have implemented
# self-checkpointing; change "when_to_transfer_output" to "ON_EXIT_OR_EVICT".
should_transfer_files = YES
when_to_transfer_output = ON_EXIT

# If you want your jobs to go on hold because they are
# running longer then expected,  uncomment this line and
# change from 24 hours to desired limit:
#periodic_hold = (JobStatus == 2) && ((CurrentTime - EnteredCurrentStatus) > (60 * 60 * 24))



# YOU SHOULD NOT NEED TO CHANGE ANYTHING BELOW THIS LINE
#
# This is a "normal" job.
universe = vanilla

# This wrapper script automates setting R or Matlab up.
executable = /home/iaross/harvesting_postscript/ChtcRun/chtcjobwrapper
requirements = (OpSysMajorVer =?= 6)

# If anything is output to standard output or standard error, 
# where should it be saved?
output = process.out
error = process.err

# Where to write a log of your jobs statuses.
log = process.log

# Tell us the version of R or Matlab you are using. Place it
# in the JobAd. Choose one. Or comment both if it is some other
# kind of program.
# Arguments to the wrapper script.  Of note is the last one, --, anything
# after this goes direct to your R, Matlab or Other code.
# This gets augmented for you by mkdag.pl. Choose R or Matlab
arguments =  --type=Other --cmdtorun=ocr_pdf.py --unique=job000009 -- input.pdf --tesseract --no-cuneiform 

# Release a job from being on hold hold after half an hour (1800 seconds), up to 4 times,
# as long as the executable could be started, the input files and initial directory
# were accessible and the user log could be created. This will help your jobs to retry
# if they happen to fail due to a computer issue (not an issue with your job)
periodic_release = (JobStatus == 5) && ((CurrentTime - EnteredCurrentStatus) > 1800) && (JobRunCount < 5) && (HoldReasonCode != 6) && (HoldReasonCode != 14) && (HoldReasonCode != 22)
#

# We don't want email about our jobs. (If you do, let us know,
# there may be some additional configuration necessary.)
notification = never

# This line is completed for you
transfer_input_files = /home/iaross/harvesting_postscript/ChtcRun/submit_1_26Mar/job000009/,/home/iaross/harvesting_postscript/ChtcRun/submit_1_26Mar/shared/

# Leave the below line commented, unless you have a specific need for
# indicating a group that is not your default.
# See: http://monitor.chtc.wisc.edu/uw_condor_usage/usage1.shtml
#+AccountingGroup = "CHTC"


queue
