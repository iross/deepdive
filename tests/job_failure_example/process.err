  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
 34 45862   34 15954    0     0  6669k      0 --:--:-- --:--:-- --:--:-- 6669k100 45862  100 45862    0     0  16.4M      0 --:--:-- --:--:-- --:--:-- 28.5M
