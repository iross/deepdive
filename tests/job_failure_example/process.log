000 (1907007.000.000) 03/26 08:30:19 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907007.000.000) 03/26 08:30:56 Job executing on host: <128.104.55.84:42557>
...
006 (1907007.000.000) 03/26 08:31:04 Image size of job updated: 7
	2  -  MemoryUsage of job (MB)
	1600  -  ResidentSetSize of job (KB)
...
006 (1907007.000.000) 03/26 08:31:14 Image size of job updated: 265676
	2  -  MemoryUsage of job (MB)
	1600  -  ResidentSetSize of job (KB)
...
005 (1907007.000.000) 03/26 08:31:14 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218565  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218565  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   635362  1000000   2990580
	   Memory (MB)          :        2     1000      1000
...
000 (1907058.000.000) 03/26 08:31:41 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907058.000.000) 03/26 08:31:45 Job executing on host: <128.105.245.149:60458>
...
006 (1907058.000.000) 03/26 08:31:54 Image size of job updated: 7
	2  -  MemoryUsage of job (MB)
	1496  -  ResidentSetSize of job (KB)
...
006 (1907058.000.000) 03/26 08:32:10 Image size of job updated: 107752
	2  -  MemoryUsage of job (MB)
	1496  -  ResidentSetSize of job (KB)
...
005 (1907058.000.000) 03/26 08:32:10 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:02, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:02, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218989  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218989  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   635363  1000000   3123742
	   Memory (MB)          :        2     1000      1000
...
000 (1907064.000.000) 03/26 08:32:23 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907064.000.000) 03/26 08:32:26 Job executing on host: <128.105.245.158:57634>
...
006 (1907064.000.000) 03/26 08:32:35 Image size of job updated: 39552
	2  -  MemoryUsage of job (MB)
	1520  -  ResidentSetSize of job (KB)
...
006 (1907064.000.000) 03/26 08:32:52 Image size of job updated: 107776
	2  -  MemoryUsage of job (MB)
	1520  -  ResidentSetSize of job (KB)
...
005 (1907064.000.000) 03/26 08:32:52 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218913  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218913  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   635363  1000000   3074286
	   Memory (MB)          :        2     1000      1000
...
000 (1907072.000.000) 03/26 08:33:11 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907072.000.000) 03/26 08:33:43 Job executing on host: <128.105.244.181:33361>
...
006 (1907072.000.000) 03/26 08:33:53 Image size of job updated: 131740
	11  -  MemoryUsage of job (MB)
	10288  -  ResidentSetSize of job (KB)
...
005 (1907072.000.000) 03/26 08:33:59 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218916  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218916  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   679363  1000000   8397082
	   Memory (MB)          :       11     1000      1000
...
000 (1907083.000.000) 03/26 08:34:14 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907083.000.000) 03/26 08:34:23 Job executing on host: <128.105.245.248:33277>
...
006 (1907083.000.000) 03/26 08:34:31 Image size of job updated: 131604
	11  -  MemoryUsage of job (MB)
	10328  -  ResidentSetSize of job (KB)
...
005 (1907083.000.000) 03/26 08:34:45 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:03, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:03, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218989  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218989  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   635363  1000000   3172592
	   Memory (MB)          :       11     1000      1000
...
000 (1907089.000.000) 03/26 08:35:01 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907089.000.000) 03/26 08:35:17 Job executing on host: <128.105.245.177:37436>
...
006 (1907089.000.000) 03/26 08:35:25 Image size of job updated: 7
	2  -  MemoryUsage of job (MB)
	1500  -  ResidentSetSize of job (KB)
...
006 (1907089.000.000) 03/26 08:35:38 Image size of job updated: 107760
	2  -  MemoryUsage of job (MB)
	1500  -  ResidentSetSize of job (KB)
...
005 (1907089.000.000) 03/26 08:35:38 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:03, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:03, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218990  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218990  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   635363  1000000   3237827
	   Memory (MB)          :        2     1000      1000
...
000 (1907102.000.000) 03/26 08:35:54 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907102.000.000) 03/26 08:35:56 Job executing on host: <128.104.55.108:57357>
...
006 (1907102.000.000) 03/26 08:36:04 Image size of job updated: 131736
	11  -  MemoryUsage of job (MB)
	10304  -  ResidentSetSize of job (KB)
...
005 (1907102.000.000) 03/26 08:36:08 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218737  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218737  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   801867  1000000   4272714
	   Memory (MB)          :       11     1000      1000
...
000 (1907111.000.000) 03/26 08:36:29 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907111.000.000) 03/26 08:36:51 Job executing on host: <128.104.55.108:57357>
...
006 (1907111.000.000) 03/26 08:37:00 Image size of job updated: 131736
	16  -  MemoryUsage of job (MB)
	15736  -  ResidentSetSize of job (KB)
...
005 (1907111.000.000) 03/26 08:37:04 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:04, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:04, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218736  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218736  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   853692  1000000   4272714
	   Memory (MB)          :       16     1000      1000
...
000 (1907124.000.000) 03/26 08:37:19 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907124.000.000) 03/26 08:37:50 Job executing on host: <128.104.55.108:57357>
...
006 (1907124.000.000) 03/26 08:37:58 Image size of job updated: 107864
	16  -  MemoryUsage of job (MB)
	15828  -  ResidentSetSize of job (KB)
...
006 (1907124.000.000) 03/26 08:38:02 Image size of job updated: 245988
	16  -  MemoryUsage of job (MB)
	15828  -  ResidentSetSize of job (KB)
...
005 (1907124.000.000) 03/26 08:38:02 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:02, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:02, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218736  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218736  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   700932  1000000   4272714
	   Memory (MB)          :       16     1000      1000
...
000 (1907135.000.000) 03/26 08:38:16 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907135.000.000) 03/26 08:39:06 Job executing on host: <128.104.55.183:40864>
...
006 (1907135.000.000) 03/26 08:39:15 Image size of job updated: 131596
	11  -  MemoryUsage of job (MB)
	10308  -  ResidentSetSize of job (KB)
...
006 (1907135.000.000) 03/26 08:39:27 Image size of job updated: 266696
	11  -  MemoryUsage of job (MB)
	10308  -  ResidentSetSize of job (KB)
...
005 (1907135.000.000) 03/26 08:39:27 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:03, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:03, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218911  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218911  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   635363  1000000   6934775
	   Memory (MB)          :       11     1000      1000
...
000 (1907152.000.000) 03/26 08:39:41 Job submitted from host: <128.104.101.254:36127>
    DAG Node: job000009+job000009
...
001 (1907152.000.000) 03/26 08:40:03 Job executing on host: <128.104.55.47:56176>
...
006 (1907152.000.000) 03/26 08:40:11 Image size of job updated: 110408
	19  -  MemoryUsage of job (MB)
	18516  -  ResidentSetSize of job (KB)
...
005 (1907152.000.000) 03/26 08:40:20 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	218908  -  Run Bytes Sent By Job
	360245  -  Run Bytes Received By Job
	218908  -  Total Bytes Sent By Job
	360245  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   656657  1000000   3037819
	   Memory (MB)          :       19     1000      1000
...
