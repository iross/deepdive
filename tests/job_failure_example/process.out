_CONDOR_ANCESTOR_3822=3850:1427118199:2696399109
_CONDOR_ANCESTOR_3850=18423:1427377202:2151094119
_CONDOR_ANCESTOR_18423=18427:1427377203:3966051
TEMP=/var/lib/condor/execute/slot1/dir_18423
_CONDOR_SCRATCH_DIR=/var/lib/condor/execute/slot1/dir_18423
_CONDOR_SLOT=slot1_7
BATCH_SYSTEM=HTCondor
TMPDIR=/var/lib/condor/execute/slot1/dir_18423
_CONDOR_CHIRP_CONFIG=/var/lib/condor/execute/slot1/dir_18423/.chirp_config
_CONDOR_JOB_PIDS=
TMP=/var/lib/condor/execute/slot1/dir_18423
_CONDOR_JOB_AD=/var/lib/condor/execute/slot1/dir_18423/.job.ad
_CONDOR_JOB_IWD=/var/lib/condor/execute/slot1/dir_18423
_CHIRP_DELAYED_UPDATE_PREFIX=Chirp*
_CONDOR_MACHINE_AD=/var/lib/condor/execute/slot1/dir_18423/.machine.ad
PATH=/usr/local/bin:/usr/kerberos/bin:/usr/bin:/bin
MachineAdTest: looking for <IsEuclid>
Euclid test returned <no>
Calling curl to fetch chtcinnerwrapper try 0
Curl error <0>
MachineAdTest: looking for <IsEuclid>
Done Running chtcinnerwrapper Result was <2048>
Run failed
